docs/UserGuide.md

```diff
@@ -348,10 +348,41 @@ DonnaFin data are saved in the hard disk automatically after any command that ch
 
 ### Editing the data file
 
-DonnaFin data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+DonnaFin's data is saved as a JSON file `[JAR file location]/data/donnafin.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 If your changes to the data file makes its format invalid, DonnaFin will discard all data and start with an empty data file at the next run.
+
+Original `donnaFin.json` format           |  Invalid `donnaFin.json` format due to direct editing
```

Comment:
there's no camel case for our .json file. should be donnafin.json, do change for the rest below as well.